lang/rust/Portfile

```diff
@@ -295,8 +314,7 @@ if { ${subport} ne ${ccwrap} } {
 }
 envs_append       LD=${ld}
 
-configure.args    --enable-vendor \
```

Comment:
Why the removal of `--enable-vendor`?

Reply:
@herbygillot because it creates the second `source.crates-io`